lib/duffel_api/resources/offer.rb

```diff
@@ -4,7 +4,7 @@
 module DuffelAPI
   module Resources
     class Offer < BaseResource
-      attr_reader :allowed_passenger_identity_document_types
+      attr_reader :supported_passenger_identity_document_types
```

Comment:
can we `alias :allowed_passenger_identity_document_types :supported_passenger_identity_document_types` ? (for "backwards compatibility")